VimeoNetworking/Sources/Models/Subscription.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  self.swift
+//  Pods
+//
+//  Created by Lim, Jennifer on 2/8/17.
+//
+//
+
+import Foundation
+
+public class Subscription: VIMModelObject
```

Comment:
I agree, but somehow if i don't specify explicitly that the `toDictionary` isn't public, i get an error that says `has no member`. 🤔 
However, for all the other properties. It just works well. :man_shrugging: